src/lib/yang/alarms.lua

```diff
@@ -1,5 +1,53 @@
 module(..., package.seeall)
 
+local data = require('lib.yang.data')
+local util = require('lib.yang.util')
+
+local state = {
+   alarm_inventory = {},
+}
+
+function get_state ()
+   return {
+      alarm_inventory = state.alarm_inventory,
+   }
```

Comment:
Yep that's true, In this specific case for the alarm-inventory I could simply return state. However, when I implement alarm-list (contains the state of alarms), it will require a transformation because my alarm-list object doesn't map one to one with the schema. That's the reason why I implemented it this way. It's a bit confusing because I'm backporting code from further developed branches, so I understand it some decisions doesn't seem obvious.

Question is if an object doesn't require any transformation is it OK to use a reference to state or would I need to return a copy of the object?